.cirrus/azure_app_verify_and_publish.sh

```diff
@@ -56,28 +57,26 @@ echo "SonarQube subchart decompressed and .tgz removed."
 # 4. Navigate back to the main offer directory
 cd ../.. # Back to azure-marketplace-k8s-app/
 
-# 5. Perform Docker login to your Azure Container Registry
-echo "5. Logging into Azure Container Registry: ${ACR_REGISTRY}..."
-if [ -z "${ACR_USERNAME}" ] || [ -z "${ACR_PASSWORD}" ]; then
-  echo "WARNING: ACR_USERNAME or ACR_PASSWORD not set. Docker login might fail or prompt for credentials."
-  docker login "${ACR_REGISTRY}"
-else
-  echo "${ACR_PASSWORD}" | docker login "${ACR_REGISTRY}" --username "${ACR_USERNAME}" --password-stdin
-fi
-echo "Docker login complete."
+
+# # 5. Push required images to the ACR_REGISTRY registry
+echo "5. Push required images to the ACR_REGISTRY registry..."
+docker tag "sonarqube:${SONARQUBE_CHART_VERSION}-enterprise" "${ACR_REGISTRY}/sonarqube:${SONARQUBE_CHART_VERSION}-enterprise"
+docker tag "bitnami/postgresql:${PSQL_VERSION}" "${ACR_REGISTRY}/bitnami/postgresql:${PSQL_VERSION}"
+docker push "${ACR_REGISTRY}/sonarqube:${SONARQUBE_CHART_VERSION}-enterprise"
+docker push "${ACR_REGISTRY}/bitnami/postgresql:${PSQL_VERSION}"
 
 # 6. Run CPA verify within the container
 echo "6. Running CPA verification (cpa verify)..."
 # The -v ./:/data mounts the current directory (azure-marketplace-k8s-app) to /data inside the container.
 # CPA commands will operate on files relative to /data.
-docker run -it --rm -v /var/run/docker.sock:/var/run/docker.sock -v "$(pwd)":/data mcr.microsoft.com/container-package-app:latest cpa verify --directory /data
+docker run --rm -v /var/run/docker.sock:/var/run/docker.sock -v "$(pwd)":/data mcr.microsoft.com/container-package-app:latest cpa verify --directory /data
 echo "CPA verification complete."
 
 # 7. Run CPA buildbundle within the container
 echo "7. Building the CPA bundle (cpa buildbundle)..."
 # This creates the .cnab directory and the bundle file (e.g., sonarqube.cnab)
 # in the current directory (mounted as /data in container).
-DOCKER_BUILDKIT=1 docker run -it --rm -v /var/run/docker.sock:/var/run/docker.sock -v "$(pwd)":/data mcr.microsoft.com/container-package-app:latest cpa buildbundle --force --directory /data
+DOCKER_BUILDKIT=1 docker run --rm -v /var/run/docker.sock:/var/run/docker.sock -v "$(pwd)":/data mcr.microsoft.com/container-package-app:latest cpa buildbundle --force --directory /data
```

Comment:
Why using `DOCKER_BUILDKIT` here but NOT on the rest of the commands? Can't we `export` it at the beginning of the script instead?